psydac/api/ast/nodes.py

```diff
@@ -742,9 +743,13 @@ def weights( self ):
         return self._weights
 
 #==============================================================================
-class LocalTensorQuadrature(ArrayNode):
-    # TODO add set_positions
+class PlusGlobalTensorQuadrature(GlobalTensorQuadrature):
+    """This class represents the quadrature points and weights in the plus boundary of an interface.
```

Comment:
you mean in the plus 'side' (ie, domain) of an interface ?

Reply:
what I meant is that in general, the 'sides' and the 'boundaries' of an interface are two different things. 
In particular they are of different dimensions: for instance if the interface is a 2D object (say, a square), the 'sides' are the 3D subdomains that share the interface, whereas the boundaries are the edges of the interface.

Reply:
ah I see, you are right, I will change that